src/goto-symex/symex_target.h

```diff
@@ -17,6 +17,9 @@ Author: Daniel Kroening, kroening@kroening.com
 class ssa_exprt;
 class symbol_exprt;
 
+/// Base class for symbolic execution. Intended as a position identifier in the
```

Comment:
What is a "position identifier?"

Reply:
I wasn't sure if we have an established term to refer to program location. To the best of my understanding `symex_targett` has a similar purpose to `const_target` which is an iterator into a list of instructions. I tried rewording the description a bit.

Reply:
Oh, I see, actually a `symex_targett` is very different (although by now I'm not even sure it is a meaningful interface anymore, because we leak out `symex_target_equationt` all over the place). A `symex_targett` is the interface of the target _container_ for symbolic execution to record its symbolic steps into. As far as I am aware, `symex_target_equationt` is the only implementation of that interface.

Reply:
I hope you won't mind if I quote/paraphrase the above for the description.

Reply:
> `symex_target_equationt` is the only implementation of that interface.

Yes, afaik.